recipes/_generate_password.rb

```diff
@@ -8,41 +10,19 @@
 
 return if node['splunk']['free_license'] && node['splunk']['node_type'] != :forwarder
 
-require 'securerandom'
-
-password_file = File.join node['splunk']['external_config_directory'], 'password'
-
-old_password = File.exist?(password_file) ? File.read(password_file) : 'changeme'
-new_password = SecureRandom.hex(36)
-
-node.run_state['cerner_splunk'] ||= {}
-node.run_state['cerner_splunk']['admin-password'] = old_password
-
-execute 'change-admin-password' do # ~FC009
-  command "#{node['splunk']['cmd']} edit user admin -password #{new_password} -roles admin -auth admin:#{old_password}"
-  environment 'HOME' => node['splunk']['home']
-  sensitive true
-end
-
-ruby_block 'update admin password in run_state' do
-  block do
-    node.run_state['cerner_splunk']['admin-password'] = new_password
-  end
+# Identify the most specific vault path that matches the current node name and type.
+password_vault_paths = node['splunk']['config']['password_secrets'] || {}
```

Comment:
Did we add this attribute to the readme?

Reply:
99d91af4e5cb1251a36c06134203de5ee99de3a8